### PR TITLE
docs(release): update issue template for TRG check

### PR DESCRIPTION
**Pull request by evegufy**
## Description

- update the issue template for checking Tractus-X Release Guidelines at release
- highlight the need to reference to a tagged state to proof TRG compliance

related to https://github.com/eclipse-tractusx/sig-release/pull/750

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by evegufy** [comment]
Hi @eclipse-tractusx/automotive-tractusx-committers, as discussed in the last committers meeting, we want to improve the documentation for TRG compliance for the upcoming 24.08 release and I updated the issue templates for that. Please review.

File: .github/ISSUE_TEMPLATE/qg-checklist.md [hunk 1]
```diff
@@ -42,15 +43,16 @@ Release Managemnet Reference Issue: <!-- Note: Add the related product RM issue
...
 
 #### TRG 4 Container
 
 - [ ] [TRG 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-01) [semantic versioning](https://semver.org/) and tagging <!-- container is tagged correctly additionally to the latest tag -->
 - [ ] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02) base image is agreed  <!-- Java, Kotlin, ... if JVM based language use base image from [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) -->
 - [ ] [TRG 4.03](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-03) image has `USER` command and Non Root Container
-- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references 
+- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references
 - [ ] [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06) separate notice file for `DockerHub` has all necessary information
+- [ ] [TRG 4.06] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) root filesystem is set to read only
```
Comment (florianrusch-zf) [inline .github/ISSUE_TEMPLATE/qg-checklist.md]: I would recommend to rephrase it like this:
```suggestion
- [ ] [TRG 4.06] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) The root file system is set to read access by default, but can be overwritten by the user
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
Just a minor recommendation

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 1 above)
Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: > I would recommend to rephrase it like this:

thank you, done

**Review by florianrusch-zf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/qg-checklist.md [hunk 2]
```diff
@@ -8,26 +8,27 @@ assignees: ''
...
-Product Owner: <!-- Note: Please search for the Product Owner of this product. -->  
-Dev SPOC: <!-- Note: Please add every contributor to this product -->  
-Helm Chart Version: <!-- Note: Please note the current Helm Chart Version to check. -->  
-App Version: <!-- Note: Please note the current App Version to check. -->  
+**Show compliance with TRGs** by referencing to a tagged link in the respective repository where possible, example: [github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs](https://github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs)
 
-Release Managemnet Reference Issue: <!-- Note: Add the related product RM issue -->  
+### Close this issue once the compliance with the TRGs has been documented
+
+Committer(s): <!-- Note: Please add every contributor to this product -->
```
Comment (tom-rm-meyer-ISST) [inline .github/ISSUE_TEMPLATE/qg-checklist.md]: Please adjust note to committer

```suggestion
Committer(s): <!-- Note: Please add every committer to this product -->
```

File: .github/ISSUE_TEMPLATE/qg-checklist.md [hunk 3]
```diff
@@ -8,26 +8,27 @@ assignees: ''
...
-We will inform you about finding and proposals in separated issues, this issue here is for the Overview of the Checks!
 
-### Please keep this issue open until QG is concluded!
+Please open and fill in this issue in your product repository to document the compliance with our Tractus-X Release Guideline (TRGs)
 
-Product Owner: <!-- Note: Please search for the Product Owner of this product. -->  
-Dev SPOC: <!-- Note: Please add every contributor to this product -->  
-Helm Chart Version: <!-- Note: Please note the current Helm Chart Version to check. -->  
-App Version: <!-- Note: Please note the current App Version to check. -->  
+**Show compliance with TRGs** by referencing to a tagged link in the respective repository where possible, example: [github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs](https://github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs)
```
Comment (tom-rm-meyer-ISST) [inline .github/ISSUE_TEMPLATE/qg-checklist.md]: While quick reading I've been confused. Would coin it a little more
```suggestion
**Show compliance with TRGs** by referencing to a tagged link in the respective repository where possible, example: TRG 1.01 (see [github.com/eclipse-tractusx/example-repo/tree/1.0.0/README.md](https://github.com/eclipse-tractusx/example-repo/tree/1.0.0/README.md))
```

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Thanks for updating. Found a few things for suggestion :)

File: .github/ISSUE_TEMPLATE/qg-checklist.md [hunk 4]
```diff
@@ -42,15 +43,16 @@ Release Managemnet Reference Issue: <!-- Note: Add the related product RM issue
...
 
 - [ ] [TRG 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-01) [semantic versioning](https://semver.org/) and tagging <!-- container is tagged correctly additionally to the latest tag -->
 - [ ] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02) base image is agreed  <!-- Java, Kotlin, ... if JVM based language use base image from [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) -->
 - [ ] [TRG 4.03](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-03) image has `USER` command and Non Root Container
-- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references 
+- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references
 - [ ] [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06) separate notice file for `DockerHub` has all necessary information
+- [ ] [TRG 4.06] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) root file system is set to read access by default, but can be overwritten by the user
```
Comment (carslen) [inline .github/ISSUE_TEMPLATE/qg-checklist.md]:
```suggestion
- [ ] [TRG 4.07] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) root file system is set to read access by default, but can be overwritten by the user
```

**Review by carslen — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 2 above)
Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: thank you, done

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 3 above)
Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: thank you, done

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 4 above)
Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: thank you, done

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, thanks for updating

**Review by Siegfriedk — APPROVED** [review]

**Pull request merged**